keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1480,16 +1497,17 @@ If we were to offer a way to lookup arbitrary other resources, or even if
 we provided selective access to just some resources, this might become
 easier. This can explored as future work.
 
-#### Access to namespace metadata
+#### Access to namespace
 
-We have general agreement to include this as a feature, but need to provide
-a concrete design.
+We have general agreement to grand CEL expression access to namespace through a newly added CEL variable `namespaceObject`.
+If the resource is cluster scoped, `namespaceObject` will be null.
 
-- Namespace labels and annotations are the most commonly needed fields not
-  already available in the resource being validated. Note that
-  namespaceSelectors already allow matches to examine namespace levels, but we
-  also have use cases that need to be able to inspects the fields in CEL
-  expressions.
+`namespaceObject` will provide access to all existing fields under namespace metadata, namespace spec and namespace status except for metadata.managedFields and metadata.ownerReferences.
+The fields could be directly accessed through `namespaceObject` variable. e.g. `namespaceObject.metadata.name` or `namespaceObject.status.phase`.
```

Comment:
super nit: Can we call this variable `objectNamespace`?  It might be just me, but I'm looking for a name that hints at the fact that this is "the namespace _of_ the object", and when I see `namespaceObject` I interpret it as "a namespace that _is_ an object"

Reply:
Here is the implementation PR: https://github.com/kubernetes/kubernetes/pull/118267 
Maybe we could discuss there and update the KEP later?